### PR TITLE
React & Redux group project - Space Travelers' Hub

### PR DESCRIPTION
**Pull request by jeanbuhendwa**
#### In this pull request, we have : 

- [ ] Create React App.
- [ ] Installed [React Redux](https://react-redux.js.org/), [Redux Logger](https://www.npmjs.com/package/redux-logger) and [React Router](https://reactrouter.com/web/guides/quick-start).
- [ ] Downloaded the [free image](https://www.flaticon.com/free-icon/planet_3212567?term=space&page=1&position=19&page=1&position=19&related_id=3212567&origin=style) for the app logo.
- [ ] Created routes and view components: rockets, missions, my profile. 
- [ ] Used <NavLink /> for the page navigation links and style active class to indicate which section/page the user is currently on (underline active navigation link).
- [ ] Created directories for all Redux state slice files: rockets and missions
- [ ] used Rockets: `https://api.spacexdata.com/v4/rockets` and the Missions: `https://api.spacexdata.com/v3/missions`
- [ ] Used useSelector() Redux Hook to select the state slices and render lists of rockets and missions in corresponding routes.
- [ ] Wrote actions and reducers for cancelling rockets/dragons and leaving missions
- [ ] Rendered UI for rockets and missions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Space Travelers' Hub] <a name="about-project"></a>
+
+**[Space Travelers' Hub]** is a capstone group project built with react js & redux
+
+## 🛠 Built With <a name="built-with"></a>
+
```
Comment (algerina) [inline README.md:30]: - [ ]  It would be nice that we try to add a live demo link for the project 😉

File: README.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (algerina) [inline README.md:150]: - [ ] The MIT license link is broken kindly work on that issue.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi team,

Great job so far! 
However, you still have minor issues to fix.

May I highlight the following: 
- [x] Your project looks awesome👏  ✔️ 
- [x] Your PR is professional and well-detailed ✔️ 
- [x] I love the fact that you collaborated to a great extend as shown by the structure of the commits messages 👏  ✔️ 
- [x] It's a good  thing that you were able to get your linter checks to pass ✔️ 




## STATUS: CHANGES REQUIRED ♻️ 
PLEASE: check the comments under the review



 
 ## Optional suggestions

 - Please check the comment under the review


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Comment by alexander16108** [comment]
Hi @jeanbuhendwa @Patfarmurs , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
 Your project looks awesome👏 ✔️
 Your PR is professional and well-detailed ✔️
 I love the fact that you collaborated to a great extend as shown by the structure of the commits messages 👏 ✔️
 It's a good thing that you were able to get your linter checks to pass ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Patfarmurs** [comment]
Thank you @alexander16108

**Pull request merged**